resources/js/common.js

```diff
@@ -283,3 +282,40 @@ function postwith (to,p) {
   document.body.removeChild(myForm) ;
 }
 
+
+function isValidDate(dateString)

```

Comment:
@fondrenlibrary 
>  actually, no particular test plan nor any integration test plan. The only test by hand plan is [...]

Maybe test plan weren't the right words but yes I meant manual instructions to be able to check that this PR works as intended without prior knowledge. So I (or others) could manually test it and contribute to build enough confidence so it could be merged quicker.

Thanks, the instructions that you provided are exactly what I needed. I hope to have the time to try them.